site/content/en/docs/tasks/run/deployment.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Run Deployment"
+linkTitle: "Deployment"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a Deployment as a Kueue-managed job.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running Deployments.
+Although Kueue does not yet support managing a Deployment as a single workload, 
+it's still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the Deployment.
+
+In this section we demonstrate how to support scheduling Deployments in Kueue based on the Plain Pod integration,
+where every Pod from a Deployment is represented as a single independent Plain Pod.
+This approach allows independent resource management for the Pods, and thus scale up and down of the Deployment.
+
+This guide is for [serving users](/docs/tasks#serving-user) that have a basic understanding of Kueue.
+For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+
+2. Follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
+to learn how to enable the `v1/pod` integration and how to configure it using the `podOptions` field.
+
+3. Kueue will run webhooks for all created pods if the pod integration is enabled. The webhook namespaceSelector could be 
```

Comment:
I think this can be dropped either. It is also part of the doc for Pod integration.

Reply:
+1